service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsAboutToSubmitControllerTest.java

```diff
@@ -31,7 +31,7 @@
 class UploadC2DocumentsAboutToSubmitControllerTest extends AbstractControllerTest {
     private static final String USER_NAME = "Emma Taylor";
     private static final ZonedDateTime ZONE_DATE_TIME = ZonedDateTime.now(ZoneId.of("Europe/London"));
-    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("h:mma, d MMMM yyyy", Locale.UK);
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("d MMMM yyyy, h:mma", Locale.UK);
```

Comment:
Could this just be replaced with a call to the DateFormatterHelper?

Reply:
Ahh I see it is used to parse from string to localdatetime. There is a method in the helper that converts from string to localdate which seems to be the end result of what you want.

Reply:
Refactored to use Time interface and helper method